src/main/java/app/daos/MovieDAO.java

```diff
@@ -166,10 +165,11 @@ public MovieDTO update(MovieDTO movieDTO) {
                     }
                 });
 
-                foundMovie.setCast(foundCastMembers);
+                foundMovie.setCast(new HashSet<>(foundCastMembers));
```

Comment:
Convert the `List<CastMember> foundCastMember` to a Set instead of converting it when calling `movie.setCast()`

Reply:
Changed.